packages/base/src/index.js

```diff
@@ -329,7 +329,30 @@ module.exports = {
       },
     ],
     'import/no-webpack-loader-syntax': 'error',
-    'import/order': 'error',
+    'import/order': [
+      'error',
+      {
+        // This means that there will always be a newline between the import
+        // groups as defined below.
+        'newlines-between': 'always',
```

Comment:
I think personally this makes it easier to distinguish external imports from internal ones. I'm open to change this however.